store/supabase.js

```diff
@@ -0,0 +1,133 @@
+// state
+export const state = () => ({
+  topics: [],
+  members: [],
+  edges: [],
+  nodes: [],
+});
+
+export const actions = {
+  async fetchData({ commit }) {
+    // hit supabase to get topics, members and mappings
+    const { data: topics, error: topicError } = await this.$supabase
+      .from("topics")
+      .select("*");
+    const { data: members, error: memberError } = await this.$supabase
+      .from("members")
+      .select("*");
+    const { data: mapping, error: mappingError } = await this.$supabase
+      .from("members_topics")
```

Comment:
why call it `mapping` here and `members_topics` in SupaBase?
Let's make it consistent.

Reply:
also `mapping` is singluar here, but `mappings` is plural in the `createNodes()` signature, which is confusing